manifests/kustomize/STRUCTURE.md

```diff
@@ -29,7 +29,7 @@ kustomize
     ├── gcp
     │   ├── cluster-scoped-resources (set ../../cluster-scoped-resources as base)
     │   └── kustomization.yaml (set ../../namespaced as base)
-    ├── aws
+    ├── gcp-dev
```

Comment:
Now we have a chance to name it properly again, why was it called `dev`?
Shall we call it `gcp-in-cluster-storage`?

Reply:
Also, I'd prefer we either remove existing `env/dev` or keep it there for a while without changing the meaning.
We can add a new `env/platform-agnostic` or sth similar for the general purpose one.

Reply:
prefer dev to let user aware it's just fit for dev.
uninstall will delete data

Reply:
It's strange we tell end user sth is fit for development. They don't develop KFP, anyway.
(uninstall in fact doesn't delete data, because persistent disks are kept by default in GCP.) The only problem is that we didn't provide a guide of how they can specify a persistence disk when they install KFP.

Reply:
imaging a company want to save cost via use local/onprem cluster for quick dev/test before port it to gcp.